src/modules/mount/main.py

```diff
@@ -61,6 +61,97 @@ def get_btrfs_subvolumes(partitions):
     return btrfs_subvolumes
 
 
+def parse_global_storage(gs_value):
+    """ Something in the chain is converting on and off to true and false.  This converts it back.
+
+    :param gs_value: The value from global storage which needs to be fixed
+    :return:
+    """
+    if gs_value is True:
+        return "on"
+    elif gs_value is False:
+        return "off"
+    else:
+        return gs_value
+
+
+def mount_zfs(root_mount_point, partition):
+    """ Mounts a zfs partition at @p root_mount_point
+
+    :param root_mount_point: The absolute path to the root of the install
+    :param partition: The partition map from global storage for this partition
+    :return:
+    """
+    # Get the list of zpools from global storage
+    zfs_pool_list = libcalamares.globalstorage.value("zfsPoolInfo")
+    if not zfs_pool_list:
+        libcalamares.utils.warning("Failed to locate zfsPoolInfo data in global storage")
+        raise Exception("Internal error mounting zfs datasets")
+
+    # Find the zpool matching this partition
+    for zfs_pool in zfs_pool_list:
+        if zfs_pool["mountpoint"] == partition["mountPoint"]:
+            pool_name = zfs_pool["poolName"]
+            ds_name = zfs_pool["dsName"]
+
+    # import the zpool
+    import_result = subprocess.run(['zpool', 'import', '-R', root_mount_point, pool_name])
```

Comment:
These calls to `subprocess.run()` should be converted to use libcalamares api -- e.g. `host_run_command()` or the like. Not because these are wrong **now**, but at some point Calamares will lose the setuid-root bit, and we will add privilege-escalation as needed on the commands that are run (rather than on the whole UI).